### DISCUSSION TITLE
Where is the settings file?

### DISCUSSION THREAD

**Discussion by raffertyuy** [microsoft/AzUrlShortener#454] [Q&A] [answered]
Hi, deployed AzUrlShortener using the "Deploy to Azure" button, I believe it connects directly to this repo, so where do I edit the `src\settings.json` file described here? https://github.com/FBoucher/AzUrlShortener/wiki/How-to-Add-a-Custom-Domain#add-the-new-custom-domain-to-the-settings-file

Cloudflare Page Rules are not working, I'm guessing this is the problem?

**Comment by jslitzkerttcu** [comment]
@raffertyuy - did you ever figure this out?  I'm stuck in the same spot.

**Reply by raffertyuy** [reply]
@jslitzkerttcu no.. :(

**Reply by fboucher** [reply, marked as answer]
Opps sorry! I completely missed this question!
@jslitzkerttcu @raffertyuy if you are running in Azure, use portal. Go to your Azure Function, in the Configuration and add/ set the customDomain.

The `settings.json` is only used when running locally from your computer. And it's missing because it's part of the GitIgnore settings for security purposes.

**Reply by raffertyuy** [reply]
> Opps sorry! I completely missed this question! @jslitzkerttcu @raffertyuy if you are running in Azure, use portal. Go to your Azure Function, in the Configuration and add/ set the customDomain.
> 
> The `settings.json` is only used when running locally from your computer. And it's missing because it's part of the GitIgnore settings for security purposes.

I need to try this again, but marking this as answered. thank you!
